docker-compose.yml

```
@@ -11,11 +11,14 @@ services:
       - docker-compose.env
 
   localstack:
-    image: localstack/localstack-full:0.12.8
+    image: localstack/localstack-full:1.4.0
     environment:
       DEFAULT_REGION: eu-west-1
-      HOSTNAME_EXTERNAL: localstack
       SERVICES: es, secretsmanager
+    networks:
+      default:
+        aliases:
+          - search-service.eu-west-1.es.localhost.localstack.cloud
     volumes:
       - "./scripts/localstack:/docker-entrypoint-initaws.d"
```

The location of init scripts has changed in v1
```suggestion
      - "./scripts/localstack:/etc/localstack/init/ready.d"
```

---

Also the parameter names in `localstack_init.sh` need to be prefixed with a slash (e.g. `/local/jwt-key`) as of 1.4, otherwise they error as being invalidly formatted